gatsby-config.js

```diff
@@ -1,6 +1,6 @@
-/* eslint-env node */
+//redirector.googlevideo.com/videoplayback?expire=1586853957&ei=JVyVXv3xGJfjkgbV0qOQBQ&ip=104.223.33.41&id=a260613bb3f6d9b2&itag=37&source=picasa&begin=0&requiressl=yes&mh=Du&mm=30&mn=sn-a5mlrnel&ms=nxu&mv=u&mvi=2&pl=17&sc=yes&ttl=transient&susc=ph&app=fife&mime=video/mp4&cnr=14&dur=1434.273&lmt=1586782374367924&mt=1586846464&sparams=expire,ei,ip,id,itag,source,requiressl,ttl,susc,app,mime,cnr,dur,lmt&sig=AJpPlLswRQIgfdQ5lyTw4MaUyYzL6sU5p9kKwtYJ1jocgr87QjZ7RwQCIQDZ2qd3BdfsqDO1itbNBlV0x8XtO2khf3Hc1YEqOcqhpA==&lsparams=mh,mm,mn,ms,mv,mvi,pl,sc&lsig=ALrAebAwRQIgIrELVT4JW3z2TWN4nOW9rJ9Ck1ZH5jTyuCvf7xLeBhICIQDFt3xVEWMsw8kVHx3SuDlhkRg3cyL5UL2hmcA9mYziig==&jparams=MTA0LjE5Mi4xMzAuMTkz&upx=TW96aWxsYS81LjAgKFgxMTsgTGludXggeDg2XzY0OyBydjo3NS4wKSBHZWNrby8yMDEwMDEwMSBGaXJlZm94Lzc1LjA=#betaHost&tr=2/* eslint-env node */
```

Comment:
could you clarify, what is this link for?

Reply:
Oh my goodness, that was an accidental paste and went over my head. I'm so sorry!